core/src/integrationTest/kotlin/com/malinskiy/marathon/scenario/CacheScenarios.kt

```diff
@@ -111,5 +106,7 @@ private fun TestBody.runMarathonWithOneTest(
 
     marathon.runAsync()
 
+    stopKoin()
```

Comment:
We need to stop Koin after each run, it may be introduced in #60, but as these tests were not run due usage of `on`, it hasn't been caught